<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WFrontWall</class>
 <widget class="QWidget" name="WFrontWall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Cantina</string>
   </property>
  </widget>
  <widget class="QPushButton" name="topBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaf1Btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Scaf1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaf2Btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Scaf2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaf3Btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Scaf3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaf4Btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Scaf4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bottomBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>301</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bottom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="floorBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>450</y>
     <width>381</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Floor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="allObjsBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>All Objects</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
